scripts/zones/Abyssea-Altepa/Zone.lua

```diff
@@ -17,6 +18,8 @@ function onZoneIn(player,prevZone)
         player:setPos(435 ,0 ,320 ,136)
     end
 
+    ResetPlayerLights(player)
```

Comment:
Will this reset lights if a player logs out then logs back in? Or accepts a tractor? I think light is supposed to persist that, at least logging out, IDK about tractor.

Reply:
Accepting Tractor will remove this effect. (Note that instead of taking you to where Tractor was cast upon you, you will instead end up back at the entry encampment of whatever Abyssea area you were in, with your Visitant status, Abyssea Lights, and mob (EXP or Gil) chain all removed as if you had just entered the area again.)

Reply:
It will reset the lights when they logout then log back in though, which it shouldn't do.

Reply:
ok, ill figure it out tommorow, ty cocosolos

Reply:
This should work like retail now, needs to be tested more, but seems right my end, and works well.